SPECS/php-sebastian-diff4/php-sebastian-diff4.spec

```diff
@@ -0,0 +1,200 @@
+# remirepo/fedora spec file for php-sebastian-diff4
+#
+# Copyright (c) 2013-2020 Remi Collet
+# License: CC-BY-SA
+# http://creativecommons.org/licenses/by-sa/4.0/
+#
+# Please, preserve the changelog entries
+#
+
+%bcond_without       tests
```

Comment:
Please remove this condition, as our build tags take care of running check section. Thanks